src/modules/home/components/card-banner/CardBanner.vue

```diff
@@ -77,27 +77,49 @@ export default defineComponent({
     eventTitle: {
       type: String,
       default: "Event Title"
+    },
+    eventDescription: {
+      type: String,
+      default: "Event Title"

```

Comment:
Should we really have default , since this component wont load if there are no props in EvnerCarousel anyway